### PR TITLE
updated for the new version Couchbase Server 4.5 and simplified deployment template 

### PR DESCRIPTION
**Pull request by cihanb**
### Contributing guide
https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md

### Changelog
*
*
*
*
*

### Description of the change

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@cihanb__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: couchbase-on-ubuntu/azuredeploy.json [hunk 1]
```diff
@@ -109,8 +103,8 @@
     },
     "sourceImageName": "b39f27a8b8c64d52b05eac6a62ebad85__Ubuntu-12_04_5_LTS-amd64-server-20150413-en-us-30GB",
     "sourceImage": "[concat('/',subscription().subscriptionId,'/services/images/',variables('sourceImageName'))]",
-    "templateBaseUrl": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/couchbase-on-ubuntu/",
-    "sharedBaseUrl": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/shared_scripts/ubuntu/",
+    "templateBaseUrl": "https://raw.githubusercontent.com/azure/azure-quickstart-templates/master/couchbase-on-ubuntu/",
```
Comment (singhkay) [inline couchbase-on-ubuntu/azuredeploy.json]: Use "Azure" please. The URLs are case sensitive

**Comment by singhkay** [comment]
@cihanb It might the case sensitive nature of the URL. "azure" instead of "Azure"

**Comment by cihanb** [comment]
corrected.

**Comment by cihanb** [comment]
tests on my side are passing with the templates. not sure why the failure in travis-ci.

**Comment by singhkay** [comment]
@cihanb The wildcard dns service being used in the CI runs into transient DNS errors from time to time which is why you saw the error. I ran the CI again and it passed, merged.

**Comment by cihanb** [comment]
Excellent. Thx

**Pull request merged**